extensions/items/standalone.mjs

```diff
@@ -950,6 +967,42 @@ class Standalone {
     }
   }
 
+  /**
+   * Count the total number of items in the given list of items, that is
+   * owned by the player.
+   * @param {Object} kwargs - kwargs.items is an array of string
+   *     representing the items.
+   *   kwargs.haveItems specifies the state to go to if the player have the
+   *     amount of specified items.
+   *   kwargs.noItems specifies the state to go to if the player doesn't have
+   *     the amount of specified items.
+   *   kwargs.amount is the minimum total count to go to "haveItems" state.
+   * @return {String} nextState - The next state.
+   */
+  async s2s_sf_haveMultiItems(srcExt, playerID, kwargs, sfInfo) {
+    let amount = kwargs.amount;
+    let items = kwargs.items;
+
+    if (!Number.isInteger(amount)) amount = 1;
+
+    const result = await this.helper.callS2sAPI('items', 'CountMultiItems', playerID, items);
+    if (typeof result.error !== 'undefined') {
+      // Extension not running?
+      console.error('items.CountMultiItems() failed, maybe items ext is not running?');
+      return FSM_ERROR;
+    }
+
+    let total = 0;
+    for (const itm in result.amounts) {
+      total += result.amounts[itm];
+    }
+    if (total >= amount) {
+      return kwargs.haveItems;
+    } else {
+      return kwargs.noItems;
+    }
```

Comment:
I think the following code is better since it is more obvious that this function always has a `return` at the end.
```javascript
return (total >= amount) ? kwargs.haveItems : kwargs.noItems;
```

Besides, these two arguments should be explicitly aliased at the beginning of the function so that the developer can see all the required arguments of `kwargs`. If the requirements of `kwargs` is changed in the future, the jsdoc may not be updated, and the new developer will suffer from inconsistency between document and code. My suggestion:
```javascript
async s2s_sf_haveMultiItems(srcExt, playerID, kwargs, sfInfo) {
  const amount = kwargs.amount ?? 1; // I doubt that we shouldn't allow implicit argument. Would be better if asked the developer of NPC to assign the amount everytime
  const {items, haveItems, noItems} = kwargs; // This syntax is so great.
  
  ... // other stuffs
}
```

Reply:
Done.